settings.nix

```
@@ -2429,7 +2429,7 @@
                   nullable (record {
                     x = required float-or-int;
                     y = required float-or-int;
-                    relative-to = required (enum ["top-left" "top-right" "bottom-left" "bottom-right"]);
+                    relative-to = required (enum ["top-left" "top-right" "top" "bottom-left" "bottom-right" "bottom" "left" "right"]);
```

can you make this the same order as in the niri sources? the diff looks fucky when `top` is inserted in the middle like this

```suggestion
                    relative-to = required (enum ["top-left" "top-right" "bottom-left" "bottom-right" "top" "bottom" "left" "right"]);
```

but yeah, completely fine otherwise. easy merge after this change